.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+---
+version: 2
+updates:
+    - package-ecosystem: github-actions
+      directory: /
+      schedule:
+          interval: daily
+
+    - package-ecosystem: pip
+      directory: /
+      schedule:
+          interval: weekly
```

Comment:
Figured this might also be something that we'd want to include. Let me know if the preference would be to add this in a separate PR and I'll pop it out and add it separately.

Reply:
Seems Fine ™️  to stay in this PR